masterlist.yaml

```
@@ -3759,6 +3759,61 @@ plugins:
           - '[Extended Dialogue Interface](https://www.nexusmods.com/fallout4/mods/27216)'
           - '[SS2 - XDI Compatibility Patch](https://www.nexusmods.com/fallout4/mods/48254)'
         condition: 'active("XDI.esm") and not active("SS2_XDI Patch.esp") and not active("SS2_XDI Patch-Basic.esp") and not active("SS2_XDI Patch-Basic_esl.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - '[Horizon V1.8](https://www.nexusmods.com/fallout4/mods/17374)'
+          - '[Sim Settlements 2 - Horizon Patch](https://www.nexusmods.com/fallout4/mods/54186)'
+        condition: 'active("Z_Horizon\.es(m|p)") and not active("SS_Addon_Horizon.esp")'
+  - name: 'SS2-Jampads2.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/48618/'
+        name: 'Jampads 2 - a Sim Settlements 2 Add-on'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - '[Horizon V1.8](https://www.nexusmods.com/fallout4/mods/17374)'
+          - '[Sim Settlements 2 - Horizon Patch](https://www.nexusmods.com/fallout4/mods/54186)'
+        condition: 'active("Z_Horizon\.es(m|p)") and not active("Z_Horizon_SS2_Jampads2.esp")'
+  - name: 'SS2_FDK_TinyLiving.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/49424/'
+        name: 'im Settlements 2 - Tiny Living'
```

The "S" got cut off here:

```suggestion
        name: 'Sim Settlements 2 - Tiny Living'
```